src/webgl/p5.Geometry.js

```diff
@@ -87,7 +87,60 @@ p5.Geometry = class Geometry {
 
     this.dirtyFlags = {};
   }
-
+  /**
+   * Removes the internal Colors of p5.Geometry.
```

Comment:
oh also, we have an extra capital in here:
```suggestion
   * Removes the internal colors of p5.Geometry.
```

Reply:
Ok gonna do ASAP, wait

Reply:
Btw nice observation 😅